qgreenland/config/layers/Glaciology/Glacier terminus positions 2000-2021/glacier_terminus.py

```diff
@@ -9,7 +9,7 @@
     ConfigLayer(
         id=id_str(start=START, end=END),
         title=f'Glacier termini {START} to {END}',
-        description=f'Glacier terminus during interval {START} to {END}.',
+        description=f'Glacier terminus during the {START}-{END} winter season.',
```

Comment:
Clarifying that the period (e.g., 2000-2001) is actually referencing the winter season (end of 2000/start of 2001)